src/guide/kotlin-java.md

```diff
@@ -11,17 +11,24 @@ Without further ado, here’s an example `build.gradle` file.
 ```kotlin
 allprojects {
     repositories {
-        ...
-        maven { url 'https://jitpack.io' }
+        maven {
+            url "https://nexus.iroha.tech/repository/maven-soramitsu/"
```

Comment:
is that working without authentication? otherwise let's use github's ones

Reply:
Yes, that's work without auth